<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>outDialog</class>
 <widget class="QDialog" name="outDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="formatBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>.mp4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.mkv</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.mov</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="scaleBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1920:1080</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1280:720</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>480:270</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="rateBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>4000k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2000k</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1000k</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>码率:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>分辨率:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>输出格式:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enterBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
